source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -4103,14 +4096,14 @@ EXP_PYMETHODDEF_DOC(KX_GameObject, rayCast,
 
 	if (dist != 0.0f) {
 		mt::vec3 toDir = toPoint - fromPoint;
-		if (mt::FuzzyZero(toDir.LengthSquared())) {
+		if (mt::FuzzyZero(toDir)) {
```

Comment:
This is changed because comparing the squared length is less precise than comparing the Manhattan distance for small vectors.